cli/modes/Client.cs

```diff
@@ -44,6 +48,32 @@ class ClientMode : Mode
     private void SecureConnection() {
         connection.AttemptSecuringConnection();
     }
+    
+    private void FileDirectory(ArraySegment<string> opts) {
+        if (opts.Count > 1) {
+            Console.WriteLine("Invalid number of arguments!");
+            return;
+        }
+        if (opts.Count == 0)
+        {
+            connection.GetFileDirectory("");
```

Comment:
Client should store a "current" directory and an argumentless ls should list files in that directory.